### PR TITLE
Code duplication removal, Made ssd1306 more like the st7735 implementation

### PR DESCRIPTION
**Pull request by itzandroidtab**
@R2D2-2019/leads

### CODE REVIEW

File: code/src/ssd1306.cpp [hunk 1]
```diff
@@ -56,4 +59,10 @@ namespace r2d2::display {
         // update the local cursor
         cursor.x++; // TODO: cursor fallthrough
     }
+
+
+    uint16_t ssd1306_i2c_c::color_to_pixel(const hwlib::color &c) {
+        // return as bool becouse we only need bools for the display
```
Comment (JasperOosterbroek) [inline code/src/ssd1306.cpp]: typo `becouse` should be `because`

File: code/src/ssd1306.cpp [hunk 2]
```diff
@@ -56,4 +59,10 @@ namespace r2d2::display {
         // update the local cursor
         cursor.x++; // TODO: cursor fallthrough
     }
+
+
+    uint16_t ssd1306_i2c_c::color_to_pixel(const hwlib::color &c) {
```
Comment (JasperOosterbroek) [inline code/src/ssd1306.cpp]: I assume this is a uint16_t because it is an inheritance, it should be explained in doxygen why it returns a bool.

Comment (JasperOosterbroek) [reply code/src/ssd1306.cpp]: in the hpp ofcourse

**Comment by itzandroidtab** [comment]
I made a issue for the typo's for the new team. So they should be able to find this pr to correct those.

**Pull request merged**